.github/workflows/preview_link.yml

```diff
@@ -17,10 +18,11 @@ jobs:
     - name: Generate links
       id: comment_body
       run: |
-        for file in ${{ steps.changed_files.outputs.added_modified }}; do
+        for file in ${{ steps.changed_files.outputs.all }}; do
```

Comment:
If we do all, won't it include deleted files?

Reply:
yes but i'd like to click on those links to check if the alias is working

Reply:
yes, you want links to deleted files, so that redirects may be tested

Reply:
Makes sense 👍